src/rgw/rgw_sal_motr.cc

```diff
@@ -2005,6 +2005,10 @@ int MotrObject::MotrDeleteOp::delete_obj(const DoutPrefixProvider* dpp, optional
           return rc;
       }
     } else {
+      if (ent.is_delete_marker()) {
```

Comment:
we should at least return the details of the existing delete-marker in the response.

Reply:
Changed.

Reply:
>> How does it work with RGW on RADOS?
It will not return any field in the response.

Reply:
@shriya-deshmukh, thanks for checking. I think it makes sense to return them. The information might be useful for the user. Do you think it will break the test?

Reply:
we can discuss further with @bkirunge7  and @cdeshmukh  about this, but so far we haven't seen any use case that might need this. If someone wants the delete marker they can simply run "ListObjectVersions" API and get the delete marker version id.
cc: @cdeshmukh , @bkirunge7, @sachitanands, @priyanka25081999

Reply:
FYI ,
Also there is ceph test suite, which we re running against RADOS/MGW/AWS.
there also I see its not expecting delete marker details in response if delete marker is already present for that object.

reference about ceph test suite : https://jts.seagate.com/browse/CORTX-30150 
test case "s3tests_boto3.functional.test_s3.test_versioning_multi_object_delete_with_marker "

Reply:
Will it break the test case if we return the information about the existing delete marker?

Reply:
yes. this test will fail

Reply:
@sjain09, @bkirunge7 - what should we do: update the test or don't return anything?

Reply:
if there is nothing returned by AWS and Ceph then we should follow the same.

Reply:
@sjain09 AWS & Ceph RGW behavior are different here. Two options here:
1. If we decide to go with AWS behavior, Ceph test also would also needed to be changed & proposed in Ceph repo too, which/when they approve, we can move ahead with this PR i believe.
2. Stay with Ceph RGW behavior.

Though going by latest discussions about these alignment issues, if fixes are in SAL layer, we should align with AWS, so option 1 above guess we should take